src/main/java/org/pac4j/vertx/handler/impl/CallbackHandler.java

```diff
@@ -51,75 +43,37 @@
     private final HttpActionAdapter httpActionHandler = new DefaultHttpActionAdapter();
     private final Vertx vertx;
     private final Config config;
+    private final boolean multiProfile;
 
     protected String defaultUrl = Pac4jConstants.DEFAULT_URL_VALUE;
```

Comment:
Right now we're not passing any config into ApplicationLogoutHandler (this may change as we extend for new 1.9.1 features, but those are outside scope of this PR) so I won't introduce ApplicationLogoutHandlerOptions yet. The only option we use in CallbackHandler is multiProfile so for now I will make that a constructor param, though again if we have additional options relating to 1.9 functionality I will introduce the Options class for it. Renamed the other to SecurityHandlerOptions.